examples/patron-list.php

```diff
@@ -62,8 +62,9 @@
         $pledge_data = $pledges_response->get('data')->get($pledge_data_key);
         $pledge_amount = $pledge_data->attribute('amount_cents');
         $patron = $pledge_data->relationship('patron')->resolve($pledges_response);
+	$patron_id = $pledge_data->relationship('patron')->get('id');
```

Comment:
Is this line actually weirdly indented is this github not parsing whitespace correctly?